packages/react-components/react-nav-preview/src/components/NavCategoryItem/useNavCategoryItem.styles.ts

```diff
@@ -1,57 +1,77 @@
-import { makeResetStyles, mergeClasses, makeStyles } from '@griffel/react';
-import { typographyStyles } from '@fluentui/react-theme';
+import { makeStyles, mergeClasses } from '@griffel/react';
 import { SlotClassNames } from '@fluentui/react-utilities';
+import { typographyStyles } from '@fluentui/react-theme';
 
 import type { NavCategoryItemSlots, NavCategoryItemState } from './NavCategoryItem.types';
+import {
+  useContentStyles,
+  useIconStyles,
+  useIndicatorStyles,
+  useRootDefaultClassName,
+} from '../sharedNavStyles.styles';
 
 export const navCategoryItemClassNames: SlotClassNames<NavCategoryItemSlots> = {
   root: 'fui-NavCategoryItem',
+  icon: 'fui-NavCategoryItem__icon',
   expandIcon: 'fui-NavCategoryItem__expandIcon',
 };
 
-/**
- * Styles for the root slot
- */
-const useRootStyles = makeResetStyles({
-  display: 'flex',
-  ...typographyStyles.body1,
-});
-
-const useContentStyles = makeStyles({
-  icon: {
-    display: 'flex',
+const useExpandIconStyles = makeStyles({
+  base: {
+    marginInlineStart: 'auto',
   },
   open: {
-    transform: 'rotate(-90deg)',
-  },
-  closed: {
     transform: 'rotate(90deg)',
   },
   selected: typographyStyles.body1Strong,
 });
 
+/**
+ * Styles for the root slot
+ */
+export const useRootStyles = makeStyles({
+  base: {
+    width: '100%',
+  },
+});
+
 /**
  * Apply styling to the NavCategoryItem slots based on the state
  */
 export const useNavCategoryItemStyles_unstable = (state: NavCategoryItemState): NavCategoryItemState => {
-  const defaultRootStyles = useRootStyles();
+  const rootStyles = useRootStyles();
+  const defaultRootClassName = useRootDefaultClassName();
   const contentStyles = useContentStyles();
+  const indicatorStyles = useIndicatorStyles();
+  const iconStyles = useIconStyles();
+  const expandIconStyles = useExpandIconStyles();
 
-  const { selected } = state;
+  const { selected, open } = state;
 
   state.root.className = mergeClasses(
     navCategoryItemClassNames.root,
-    defaultRootStyles,
+    defaultRootClassName,
+    rootStyles.base,
     state.root.className,
```

Comment:
state.root.className should be the last item so that users can always override previous styles

Reply:
good catch - thanks!